libs-haskell/da-hs-base/src/DA/Service/Logger/Impl/IO.hs

```diff
@@ -106,8 +107,5 @@ showTags :: [T.Text] -> String
 showTags tags = " [" <> unwords (reverse $ map T.unpack tags) <> "] "
 
 prioToString :: Priority -> String
-prioToString = \case
-      Error   -> " [ERROR] "
-      Info    -> " [INFO]  "
-      Debug   -> " [DEBUG] "
-      Warning -> " [WARN]  "
+prioToString prio = " [" ++ prio' ++ "] "
```

Comment:
There's a slight change in spacing after the closing bracket here. Let's hope nobody is relying on this. :crossed_fingers: